ccloud-monitoring/docs/producer-scenarios/connectivity-problem.rst

```diff
@@ -0,0 +1,89 @@
+.. _ccloud-monitoring-producer-connectivity-problem:
+
+Connectivity Problem
+********************
+
+In the ``producer`` container, add a rule blocking network traffic that has a destination TCP port ``9092``. This will prevent the producer from reaching the broker in |ccloud|.
+
+This scenario will look at |ccloud| metrics from the Metrics API and client metrics from the client application's MBean object ``kafka.producer:type=producer-metrics,client-id=producer-1``.
+
+Introduce failure scenario
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Add a rule blocking traffic in the ``producer`` container on port ``9092`` which is used to talk to the broker:
+
+   .. code-block:: bash
+
+      docker-compose exec producer iptables -A OUTPUT -p tcp --dport 9092 -j DROP
+
+Diagnose the problem
+^^^^^^^^^^^^^^^^^^^^
+
+#. Open `Grafana <localhost:3000>`__ and login with the username ``admin`` and password ``password``.
+
+#. Navigate to the ``Producer Client Metrics`` dashboard. Within a minute you should see a downward
+   trend in outgoing bytes which can be found by the expanding the ``Throughput`` tab.
+   Within two minutes, the top level panels like ``Record error rate`` (derived from |ak| MBean attribute ``record-error-rate``) should turn red, a major indication something is wrong.
+   You should see the spark line in the ``Free buffer space`` (derived from |ak| MBean attribute ``buffer-available-bytes``) panel go down and a bump in ``Retry rate`` (derived from |ak| MBean attribute ``record-retry-rate``)
+   This means the producer is not producing data, which could happen for a few reasons.
+
+
+   |Producer Connectivity Loss|
+
+
+#. In order to say this is a truly a problem on the producer end, check the status of the |ccloud| cluster, specifically that it is accepting requests. Navigate to the ``Confluent Cloud`` dashboard.
+
+#. Look at the top panels, they should all be green which means the cluster is operating safely within its resources.
+
+   |Confluent Cloud Panel|
+
+   For a connectivity problem in a client, look specifically at the ``Requests (rate)``. If this value
```

Comment:
There is a lot of incredibly useful information in this paragraph but it gets lost as a single paragraph.  WDYT about splitting each point/metric out into (1) requests rate limit (2) throttling

Reply:
Ack. I split up some content here into either bullet points or separate paragraphs, then validated the build locally.